agora_analytica/data/interpolation/combine.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/pin/python
+import pandas as pd
+import numpy as np
+import array
+import os.path
+import json
+from ..utils import instance_path
+from ..scrape import hae_dataa
+
+
+instancepath = instance_path()
+try:
+    instancepath = instance_path()
+    with open(os.path.join(instancepath, f"parties2.json"), "r",encoding='utf-8') as json_file:
+        parties = json.load(json_file)
+
+    with open(os.path.join(instancepath, f"constituencies.json"), "r",encoding='utf-8') as json_file:
+        constituencies = json.load(json_file)
+except FileNotFoundError:
+    text = input("Seems like you are missing some data. Do you want to scrape now, THIS MIGHT TAKE UP TO 15 minutes Y/N? ") 
+    if (text == "Y"):
+        hae_dataa()
+    else:
+        raise Exception("Missing data")
+
+
+j = 1
+constituentArray = []
+constituentArray.append("")
+while j <= 14:
```

Comment:
Kommentteihin tai johonkin muuttujaan miksi se on `14` - se ei ole itsestäänselvää.

Pythonissa lisäksi suositaan "funktionaalisempaa" looppia, ettei tarvitse huolehtia siitä iteraattorin kasvatuksesta.
```suggestion
for(i in range(14)):
```